CHANGELOG.md

```diff
@@ -13,6 +14,7 @@
 
 ## Breaking changes
 
+- Renamed "electrode diffusivity" to "particle diffusivity" as a non-breaking change with a deprecation warning ([#3624](https://github.com/pybamm-team/PyBaMM/pull/3624))
```

Comment:
This says it is a non-breaking change, but it is listed under breaking changes

Reply:
Thank you for reviewing. (I just deleted the line.)